src/Data/String/Gen.purs

```diff
@@ -29,3 +29,16 @@ genAsciiString' = genString CG.genAsciiChar'
 -- | Generates a string made up of numeric digits.
 genDigitString :: forall m. MonadRec m => MonadGen m => m String
 genDigitString = genString CG.genDigitChar
+
+-- | Generates a string using characters from the English alphabet.
```

Comment:
How about "basic Latin" rather than "English", per the `Char` gens / usual ISO nomenclature?

Reply:
Didn't know this was ISO nomenclature. I found it surprising because in my mind, "basic Latin" doesn't include the letter "u," for example. I will change it.

Reply:
Yeah, that's a good point actually! But https://en.wikipedia.org/wiki/ISO_basic_Latin_alphabet